### ISSUE TITLE
Migrate annotations to attribute in 30 set

### ISSUE THREAD

**Issue by come-nc** [nextcloud-libraries/rector#17]
From upgrade to 30 documentation:

- Using the `@PasswordConfirmationRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\PasswordConfirmationRequired] attribute should be used instead.
- Using the `@CORS` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\CORS] attribute should be used instead.
- Using the `@PublicPage` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\PublicPage] attribute should be used instead.
- Using the `@ExAppRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\ExAppRequired] attribute should be used instead.
- Using the `@AuthorizedAdminSetting` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\AuthorizedAdminSetting] attribute should be used instead.
- Using the `@SubAdminRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\SubAdminRequired] attribute should be used instead.
- Using the `@NoAdminRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\NoAdminRequired] attribute should be used instead.
- Using the `@StrictCookieRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\StrictCookiesRequired] attribute should be used instead.
- Using the `@NoCSRFRequired` annotation is deprecated and the #[OCP\AppFramework\Http\Attribute\NoCSRFRequired] attribute should be used instead.

**Comment by provokateurin** [comment]
Technically they have been deprecated way earlier in the moment when we added the attributes. I only added some debugging logging in case the annotations are still used and took the time to also not that down in the changelog.
For none of the attributes you need 30, so we can move them to some earlier sets (though they might not all be in the same).

**Comment by come-nc** [comment]
Yeah can be done in the version adding the attribute, that works as well.

**Comment by provokateurin** [comment]
Would be better because if it was in the 30 set we could only upgrade to them next year while they are already available on all supported versions.
